### PR TITLE
DOPS-877: Update SIS-SDK with latest SIS domain model changes

### PR DESCRIPTION
**Pull request by idstef**

### CODE REVIEW

**Comment by nikolaposa** [comment]
@idstef Can you please run `composer cs-fix` before merging this one? I see that there are lot of files changed for no reason (i.e. trailing spaces, newlines). It will be very hard to tracking actual, valuable changes otherwise.

**Review by zoranarbor — DISMISSED** [review]

**Comment by idstef** [comment]
@nikolaposa Sure. I'll merge after I apply cs-fix changes

**Review by nikolaposa — APPROVED** [review]

**Pull request merged**
